3-platform/shopping-cart/infra/src/main/java/com/_6core/platform/shopping/cart/infra/adapter/driven/persistence/ShoppingCartRepository.java

```diff
@@ -0,0 +1,10 @@
+package com._6core.platform.shopping.cart.infra.adapter.driven.persistence;
+
+import com._6core.platform.shopping.cart.infra.adapter.driven.persistence.entity.ShoppingCartEntity;
+import reactor.core.publisher.Mono;
+
+public interface ShoppingCartRepository {
+  Mono<ShoppingCartEntity> getCart(String cartId);
+
+  Mono<Void> cleanCart(String cartId);
```

Comment:
never return the Void/Null/void 

create some Class that will be response of the ???ShoppingCartPort